ui/App/OrgScreen/OrgHeader.svelte

```diff
@@ -144,14 +144,14 @@
               <CopyableIdentifier value={seedId} kind="seedAddress" />
             </div>
           {/if}
-          {#if seedApi}
+          {#if seedHost}
             <div class="row">
               <Icon.Globe />
               <div class="url">
                 <span
                   on:click={() => {
-                    seedApi && ipc.openUrl(seedApi);
-                  }}>{seedApi}</span>
+                    seedHost && ipc.openUrl(seedHost);
```

Comment:
`seedHost` is not a URL per se, but if you want, you can open `https://{seedHost}:8777` which is the JSON REST API..

Reply:
Though actually maybe it's best to not create a link out of it.

Reply:
Yeah that was my comment on this PR, I think the link is of no good use here.
But eventually we could use the `CopyableIdentifier` as being used for the seed id.
What do you think?

Reply:
Yes on the web client we expose the full `{id}@{host}:8776`, as that's useful. I'd let the upstream devs decide on this though.

Reply:
I think using just the copyable identifier is fine. Showing the device URL to easily add it to the seed nodes also makes sense. Any thoughs on this @juliendonck and @rudolfs?

Reply:
Is `8776` the default port for the p2p protocol? If so, showing `{id}@{host}:8776` would make sense.

What I intended to show in the header was the seed address, e.g. for our current seed host:
`hynkyndc6w3p8urucakobzna7sxwgcqny7xxtw88dtx3pkf7m3nrzc@sprout.radicle.xyz:12345`, so that a user could copy that and add it directly to their seed list on the Network screen.

For this purpose we have the `<Identifier kind="seedAddress" />` component.
It looks like this when rendered:

<img width="506" alt="Screenshot 2021-09-27 at 10 54 25" src="https://user-images.githubusercontent.com/158411/134876611-bac6dc8a-b294-4f41-aa9a-727f00e4785d.png">

Reply:
So basically this:

```diff
diff --git a/ui/App/OrgScreen/OrgHeader.svelte b/ui/App/OrgScreen/OrgHeader.svelte
index 65524767..5863b4d1 100644
--- a/ui/App/OrgScreen/OrgHeader.svelte
+++ b/ui/App/OrgScreen/OrgHeader.svelte
@@ -102,7 +102,7 @@
           </div>
         {/if}
       </div>
-      {#if websiteUrl || githubUrl || twitterUrl || seedId || seedHost}
+      {#if websiteUrl || githubUrl || twitterUrl || (seedId && seedHost)}
         <div>
           {#if websiteUrl}
             <div class="row">
@@ -139,14 +139,11 @@
           {/if}
         </div>
         <div>
-          {#if seedId}
+          {#if seedId && seedHost}
             <div class="row">
-              <CopyableIdentifier value={seedId} kind="seedAddress" />
-            </div>
-          {/if}
-          {#if seedHost}
-            <div class="row">
-              <CopyableIdentifier value={seedHost} kind="seedHost" />
+              <CopyableIdentifier
+                value={`${seedId}@${seedHost}:8667`}
+                kind="seedAddress" />
             </div>
           {/if}
         </div>
```

<img width="1552" alt="Screenshot 2021-09-27 at 11 00 17" src="https://user-images.githubusercontent.com/158411/134877696-89dae054-feeb-4e2f-bcb4-c557b02744f7.png">

Reply:
Okay no problem! Sounds great!

Reply:
Done